<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAEWindow</class>
 <widget class="QMainWindow" name="DAEWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dagor Asset Explorer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Asset tree</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomTreeView" name="treeView">
          <property name="dragDropMode">
           <enum>QAbstractItemView::DropOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="MapTab" name="mapTab">
       <attribute name="title">
        <string>Map prop layout exporter</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionOpenFiles"/>
    <addaction name="separator"/>
    <addaction name="actionUnmount"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCollapse"/>
    <addaction name="actionExpand"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open asset folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="text">
    <string>Collapse all</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="text">
    <string>Expand all</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+0</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>Open assets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionUnmount">
   <property name="text">
    <string>Unmount all assets</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionOpenMap">
   <property name="text">
    <string>Open level file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeView</class>
   <extends>QTreeView</extends>
   <header>gui.customtreeview</header>
  </customwidget>
  <customwidget>
   <class>MapTab</class>
   <extends>QWidget</extends>
   <header>gui.mapDialog</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
